cypress/support/hub-commands.ts

```diff
@@ -305,24 +305,48 @@ Cypress.Commands.add('deleteRemoteRegistry', (remoteRegistryId: string) => {
   cy.requestDelete(hubAPI`/_ui/v1/execution-environments/registries/${remoteRegistryId}/`);
 });
 
+Cypress.Commands.add('deleteCollection', (collection: string, namespace: string) => {
+  cy.galaxykit(`collection delete ${namespace} ${collection}`);
+});
+
+Cypress.Commands.add('uploadCollection', (collection: string, namespace: string) => {
+  cy.galaxykit(`collection upload ${namespace} ${collection}`);
+});
+
 Cypress.Commands.add(
-  'deleteCollection',
-  (
-    collectionName: string,
-    namespaceName: string,
-    repository: string,
-    options?: {
-      /** Whether to fail on response codes other than 2xx and 3xx */
-      failOnStatusCode?: boolean;
-    }
-  ) => {
-    cy.requestDelete(
-      hubAPI`/v3/plugin/ansible/content/${repository}/collections/index/${namespaceName}/${collectionName}/`,
-      options
-    );
+  'approveCollection',
+  (collection: string, namespace: string, version: string) => {
+    cy.galaxykit(`collection move ${namespace} ${collection} ${version} staging published`);
   }
 );
 
-Cypress.Commands.add('uploadCollection', (collection: string, namespace: string) => {
-  cy.galaxykit(`-i collection upload ${namespace} ${collection}`);
+Cypress.Commands.add('collectionCopyVersionToRepositories', (collection: string) => {
+  cy.navigateTo('hub', 'collections');
+  cy.filterTableByText(collection);
+
+  cy.get('.pf-v5-c-data-list__item-content').then(() => {
+    cy.get('[data-cy="data-list-name"]').should('have.text', collection);
+  });
+  cy.get('.pf-v5-c-data-list__item-action').within(() => {
```

Comment:
is there a data-cy locator available here? we try very hard to avoid using patternfly specific locators, as they can change anytime.